torchinductor/codegen/cpp.py

```diff
@@ -194,6 +194,10 @@ def sqrt(x):
     def rsqrt(x):
         return f"1 / std::sqrt({x})"
 
+    @staticmethod
+    def signbit(x):
```

Comment:
I don't see how this would ever be called given that we don't have lowering for signbit

Reply:
I think you are missing a `register_pointwise(aten.signbit)` line.